src/shogun/distributions/Gaussian.cpp

```diff
@@ -267,7 +267,7 @@ float64_t CGaussian::compute_log_PDF(SGVector<float64_t> point)
 	return -0.5 * answer;
 }
 
-SGVector<float64_t> CGaussian::get_mean()
+SGVector<float64_t>& CGaussian::get_mean()
```

Comment:
we need use linalg add function in https://github.com/shogun-toolbox/shogun/pull/3859/files#diff-b59cfc6c549dd52160d0ce73b6356b04R377